sdk/core/core/CMakeLists.txt

```diff
@@ -45,6 +45,8 @@ add_library (
 
 if(MSVC)
   target_compile_options(az_core PRIVATE /Wall /WX /wd5031 /wd4668 /wd4820 /wd4255 /analyze)
```

Comment:
/WX and -Werror are still annoying to me. you need to be able to opt out of them to support package maintainers and people using newer compilers than you.